ReSharper.FSharp/src/FSharp.Common/src/Util/FSharpXmlDocService.fs

```diff
@@ -13,9 +17,26 @@ open JetBrains.ReSharper.Psi.XmlIndex
 open JetBrains.UI.RichText
 open JetBrains.Util
 
+[<ShellComponent>]
+type FSharpXmlDocHtmlPresenter(theming: ITheming, factory: XmlDocSectionFactory) =
+    inherit XmlDocHtmlPresenter(theming, factory)
+
+    let crefManager =
+        { new CrefManager() with
+            member x.Process(cref, _, _, _, _) = XmlDocPresenterUtil.ProcessCref(cref)
+            member x.Create _ = null }
+
+    member x.Run(node: XmlNode) =
+        let result = RichText()
+        let declaredElement = null.As<DeclaredElementInstance>()
+        XmlDocHtmlPresenter
+            .ConvertProcessor(node, null, declaredElement, false, FSharpLanguage.Instance, crefManager, factory, theming)
```

Comment:
Why one `null` is passed as is and another one is named as `declaredElement` (which strictly is a wrong name, since it's a declared element instance)?